### DISCUSSION TITLE
A installer that applies Dracula Theme in all installed apps automatically

### DISCUSSION THREAD

**Discussion by itallolobo** [dracula/dracula-theme#770] [Ideas] [answered]
I love Dracula theme and I use it in every single program it has support for. But the thing I feel annoying is to set the theme manually in every single app, A program that detects all the supported apps and applies the Dracula theme would be a time saver, especially for apps that require extra steps to add custom themes. I really think that the community would like something like that.

**Comment by luxonauta** [comment, marked as answer]
It would be fantastic; it's a great suggestion. 🎯

However, at the moment, we can't dedicate time to build something like this; the 🪨 complexity would be enormous since each app and service has its particularities, and there are already more than 200 😅!

✅ That said, we are always open to contributions of any kind, from ideas to software, that we can include in the organization to make life easier for our community.
